Formula/v/vulkancapsviewer.rb

```diff
@@ -0,0 +1,30 @@
+class Vulkancapsviewer < Formula
+  desc "Client application to display hardware implementation details for Vulkan GPUs"
+  homepage "https://github.com/SaschaWillems/VulkanCapsViewer"
+  url "https://github.com/SaschaWillems/VulkanCapsViewer.git", tag: "3.32", revision: "3024e3148a7b47706b6d259f8e0ad0a2aa651330"
+  license "LGPL-3.0-or-later"
+
+  depends_on "cmake" => :build
+  depends_on "vulkan-headers" => :build
+  depends_on "qt@5"
+  depends_on "vulkan-loader"
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args,
+                    "-DQt5_DIR=#{Formula["qt5"].opt_lib}/cmake/Qt5",
+                    "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+                    "-DVULKAN_LOADER_INSTALL_DIR=#{Formula["vulkan-loader"].opt_prefix}"
+    system "cmake", "--build", "build"
+    if OS.mac?
+      bin.install "build/vulkanCapsViewer.app/Contents/MacOS/vulkanCapsViewer"
```

Comment:
No `cmake --install`?

Reply:
cmake configuration here would just try to install this as an app bundle, which I think is inappropriate because:
* This accepts command line inputs.
* Produces useful output on stdout.
* Can be indirectly modified by environment variables.

I don't see a way to have cmake ignore the bundle property, so this is a simple way to get this done.

Reply:
Can you try and report that as an issue upstream? I'm sure Homebrew won't be the last to prefer a cli

Reply:
Yeah I will, though I am still researching what would be the best way forward to handle this.
This might even be an improvement we can suggest to cmake itself.

It seems to me that for the specific case of homebrew formulas on MacOS, we would never want to bundle apps, so this could in effect be a special MacOS platform where the MACOSX_BUNDLE property should not be honored.

For this specific upstream, it would be harder to make a recommendation to just drop the bundle, because otherwise they release one themselves, but this is a non-standard bundle which doesn't pack all of it's dependent dylibs and relies on the VulkanSDK bundle being installed.

VulkanSDK is not distributed as a cask in homebrew, the VulkanSDK components are distributed as individual formulas instead, and they are not interchangeable due to path differences.

Reply:
Where is the app bundle installed to by `cmake --install`?

Reply:
To `<PREFIX>/bin`.